### PR TITLE
New Files added: Assignment6 and Project_report

### PR DESCRIPTION
**Pull request by vishnukopnar**

### CODE REVIEW

File: Vishnu_45/html/Ass6/assignment1.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      <input type="text" name="lastname" size="15" /> <br> <br>
+      <label>
+        Gender :
+      </label>
+      <input type="radio" name="male" /> Male
+      <input type="radio" name="female" /> Female
+      <input type="radio" name="other" /> Other
+      <br>
+      <br>
+
```
Comment (Pranjali1999) [inline Vishnu_45/html/Ass6/assignment1.html]: Give proper indentation and remove extra lines.

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

File: Vishnu_45/html/Ass6/assignment1.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE Html>
+  <head>
+    <title>
+      Page
```
Comment (shivnanda6272) [inline Vishnu_45/html/Ass6/assignment1.html]: do proper indentation

File: Vishnu_45/html/Ass6/assignment1.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  <style>
+    h1 {
+    background-color: rgba(255, 255, 255, 0.925);
+  }
+  h2 {
+    background-color: rgba(255, 255, 255, 0.925);
+  }
+  </style>
+  <body bgcolor="Lightskyblue">
+
```
Comment (shivnanda6272) [inline Vishnu_45/html/Ass6/assignment1.html]: remove extra lines

File: Vishnu_45/html/Ass6/assignment1.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      </label>
+      <input type="checkbox" name="HTML" /> HTML
+      <input type="checkbox" name="Java Script" /> Java Script 
+      <input type="checkbox" name="Java" /> Java
+      <input type="checkbox" name="Oracle" /> Oracle
+      <br>
+      <br>
+    </form>
+  </body>
+</html>
```
Comment (shivnanda6272) [inline Vishnu_45/html/Ass6/assignment1.html:46]: do proper indentation , remove all extra line and add extra  end line...

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]
make changes proper

**Review by Akash-ing — APPROVED** [review]

**Review by shivnanda6272 — APPROVED** [review]

File: Vishnu_45/html/Ass6/assignment1.html [hunk 5]
```diff
@@ -1,59 +1,46 @@
...
+    </head>
+    <style>
+        h1 {
+            background-color: rgba(255, 255, 255, 0.925);
+        }
+        h2 {
+            background-color: rgba(255, 255, 255, 0.925);
+        }
+    </style>
+    <body bgcolor="Lightskyblue">
```
Comment (Pranjali1999) [inline Vishnu_45/html/Ass6/assignment1.html]: bgcolor is outdated.

File: Vishnu_45/html/Ass6/assignment1.html [hunk 6]
```diff
@@ -1,59 +1,46 @@
...
+        <form>
+            <header class="header">
```
Comment (Pranjali1999) [inline Vishnu_45/html/Ass6/assignment1.html:16]: here is no need to header tag, so remove it.

File: Vishnu_45/html/Ass6/assignment1.html [hunk 7]
```diff
@@ -1,59 +1,46 @@
...
+                <h1 align="right">MI-9-Intern HTML Assignment</h1>
+            </header>
+            <br />
+            <hr />
+            <header class="header">
+                <h2 align="right">Personal Details</h2>
```
Comment (Pranjali1999) [inline Vishnu_45/html/Ass6/assignment1.html:22]: Here  is no need to header tag, so remove it. & align is also out dated. so, remove from every where.

File: Vishnu_45/html/Ass6/assignment1.html [hunk 8]
```diff
@@ -1,59 +1,46 @@
...
+            <input type="text" name="lastname" size="15" /> <br />
+            <br />
+            <label> Gender : </label>
+            <input type="radio" name="male" /> Male <input type="radio" name="female" /> Female <input type="radio" name="other" /> Other
+            <br />
+            <br />
+            <label> Age : </label>
+            <input type="radio" name="number" /> less than 18 <input type="radio" name="number" /> 18-25 <input type="radio" name="number" /> 26-40
+            <header class="header">
+                <h2 align="right">Academic Details</h2>
```
Comment (Pranjali1999) [inline Vishnu_45/html/Ass6/assignment1.html:37]: Here  is no need to header tag, so remove it. & align is also out dated. so, remove from every where.

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

**Review by vinod2406 — APPROVED** [review]
ok

**Review by akash19972306 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

File: Vishnu_45/html/Ass6/assignment1.html [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+        <title>Page</title>
+    </head>
+    <style>
+        h1 {
+            background-color: rgba(255, 255, 255, 0.925);
+        }
+        h2 {
+            background-color: rgba(255, 255, 255, 0.925);
+        }
+    </style>
```
Comment (Pawan1005) [inline Vishnu_45/html/Ass6/assignment1.html:13]: add this in head tag

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**